### PR TITLE
changed the generic cells property

### LINKED ISSUE #1378: get_structure() for sphinx jobs gives error
**Summary**

running get_structure() for sphinx job gives error (IndexError: ('cell at step ', 25, ' not found')).  Sphinx Job is a DFT relaxation job. and cell typically doesn't change for each step.

```python
job = pr.load('sphinx_job')
job.get_structure(i) # i>0 gives the error.
```

### PR DESCRIPTION
**Pull request by skatnagallu**
Changed the cells property to repeat based on number of steps to fix issue #1378

### CODE REVIEW

**Comment by skatnagallu** [comment]
Unittest fails with AttributeError: energy_free. I am not sure why

File: pyiron_atomistics/sphinx/base.py [hunk 1]
```diff
@@ -2147,7 +2147,7 @@ def get_last(arr):
         if "positions" not in self.generic.list_nodes():
             self.generic.positions = np.array([self._job.structure.positions])
         if "cells" not in self.generic.list_nodes():
-            self.generic.cells = np.array([self._job.structure.cell.tolist()])
+            self.generic.cells = np.tile(np.array([self._job.structure.cell.tolist()]),(len(self.generic.energy_tot),1,1))
```
Comment (pmrv) [inline pyiron_atomistics/sphinx/base.py]:
```suggestion
np.tile(self._job.structure.cell,(len(self.generic.energy_tot),1,1))
```
This shouldn't have an effect, right?

Comment (skatnagallu) [reply pyiron_atomistics/sphinx/base.py]: prolly not. But the unittest still fails.

**Comment by freyso** [comment]
I think this approach is suboptimal. 

The relaxHist.sx output replicates the cell, so it should be parsable (and it actually seems to be parsed into "output.cell" rather than ...".cells" (with s), see sphinx/output_parsers.py lines 192-196. I'd rather rename the parsing target in the parser, because then we would be ready for future implementations of automatic cell optimization.

**Comment by samwaseda** [comment]
Based on @freyso's comment, I suggest to copy `generic.cell`, which will probably also solve the problem with CI

File: pyiron_atomistics/sphinx/base.py [hunk 2]
```diff
@@ -2147,7 +2147,8 @@ def get_last(arr):
         if "positions" not in self.generic.list_nodes():
             self.generic.positions = np.array([self._job.structure.positions])
         if "cells" not in self.generic.list_nodes():
```
Comment (samwaseda) [inline pyiron_atomistics/sphinx/base.py]:
```suggestion
        if "cells" not in self.generic.list_nodes() and "cell" in self.generic.list_nodes():
            self.generic.cells = self.generic.cell
```

File: pyiron_atomistics/sphinx/base.py [hunk 3]
```diff
@@ -2146,8 +2146,10 @@ def get_last(arr):
             self.generic.energy_pot = self.generic.dft.energy_free
         if "positions" not in self.generic.list_nodes():
             self.generic.positions = np.array([self._job.structure.positions])
-        if "cells" not in self.generic.list_nodes():
-            self.generic.cells = np.array([self._job.structure.cell.tolist()])
+        if "cells" not in self.generic.list_nodes() and "cell" in self.generic.list_nodes():
+            self.generic.cells = self.generic.cell
+            self.generic.cells = np.tile(np.array([self._job.structure.cell.tolist()]),
```
Comment (samwaseda) [inline pyiron_atomistics/sphinx/base.py:2151]:
```suggestion
```

Sorry apparently I missed these lines

File: pyiron_atomistics/sphinx/base.py [hunk 4]
```diff
@@ -2146,8 +2146,10 @@ def get_last(arr):
...
+                                         (len(self.generic.dft.energy_free),1,1))
```
Comment (samwaseda) [inline pyiron_atomistics/sphinx/base.py]:
```suggestion
```

Sorry apparently I missed these lines

**Comment by skatnagallu** [comment]
Shouldn't we change the key to 'cells'[ here ](https://github.com/pyiron/pyiron_atomistics/blob/0602a398b0d7149e44f6be27cc37c524a584f67c/pyiron_atomistics/sphinx/output_parser.py#L198) ?

**Comment by samwaseda** [comment]
> Shouldn't we change the key to 'cells'[ here ](https://github.com/pyiron/pyiron_atomistics/blob/0602a398b0d7149e44f6be27cc37c524a584f67c/pyiron_atomistics/sphinx/output_parser.py#L198) ?

I guess we should discuss it at the pyiron meeting. For now if the tests pass we can live with what we get here.

**Comment by skatnagallu** [comment]
can be merged now

**Review by samwaseda — APPROVED** [review]
As soon as Black runs I'm fine with merging it

**Pull request merged**